edgecompute/examples/authentication/jwt-signing/README.md

```diff
@@ -0,0 +1,21 @@
+# JWT Signing
+
+_Keyword(s):_ jwt, authentication<br>
+
+## Copyright Notice
+
+    (c) Copyright 2024 Akamai Technologies, Inc. Licensed under Apache 2 license.
+
+This project provides an Akamai Edgeworker solution for signing JSON Web Tokens (JWTs) and attaching them to API requests. It includes an example of how to sign a JWT using HMAC-SHA256 and then add the JWT and an API key to the headers of an outgoing request.
+
+## Limitations
+
+-   Secrets are stored in [Property Manager variables](https://techdocs.akamai.com/property-mgr/docs/user-defined-vars). Property Manager variables should be marked as hidden to prevent exposure with akamai-x-get-extracted-values. Depending on your security considerations, you may wish to store secrets elsewhere or use asymmetric keys.
```

Comment:
In terms of usability, it might be a good idea to explicitly name the PMVars that the user needs to set, and tell them how to get data in there. Generating keys can be a pain, even with SubtleCrypto.

Reply:
@evan-hughes can you clarify a little more? Are you saying we should provide them instruction on how to generate keys for use in this solution? Or are you saying that it should call out what explicit values are needed for signing? I can add some more context but it might be venturing a little too far into the weeds, especially given that links to resources for both the web crypto api documentation as well as the EW jwt documentation are included below.

Reply:
> Are you saying we should provide them instruction on how to generate keys for use in this solution?

If we have a new user who isn't familiar with crypto stuff, they won't know how to generate an HMAC key, export it, and encode it appropriately. 

I'd lean towards something like 
> This EdgeWorker requires you to create a Property Manager variable named PMUSER_KEY_NAME_OR_WHATEVER, set it to hidden, and then paste an encoded key into it. You can generate a key by running the following in a browser:
> ```javascript
> let key = await crypto.subtle.generateKey(
>  {
>    name: "HMAC",
>    hash: { name: "SHA-256" },
>  },
>  true,
>  ["sign", "verify"],
> );
> console.log("%s", JSON.stringify(await crypto.subtle.exportKey('jwk', key)));
> ``` 
> The string written to the console can be pasted directly into the Property Manager variable editor. 

That does switch the key encoding from `raw` to `jwk`.

Reply:
Thanks @evan-hughes, changes made along with some additional details.